TELCODOCS-1237 release notes (#63593)
openshift/openshift-docs: release_notes/ocp-4-14-release-notes.adoc

==== Update to HAProxy 2.6
With this release, {product-title} updated to HAProxy 2.6.

[id="ocp-4-14-networking-egress-service"]
==== Egress service resource to manage egress traffic for pods behind a load balancer (Technology Preview)

With this update, you can use an `EgressService` custom resource (CR) to manage egress traffic for pods behind a load balancer service. This is available as a Technology Preview feature.

You can use the `EgressService` CR to manage egress traffic in the following ways:

> * Assign a load balancer service's IP address as the source IP address for egress traffic for pods behind the load balancer service.

Reviewer: `Assign a` -> `Assign the`?
`for egress traffic for` -> `of egress traffic for` / `for egress traffic of` ?

[id="ocp-4-14-networking-egress-service"]
==== Egress service resource to manage egress traffic for pods behind a load balancer (Technology Preview)

With this update, you can use an `EgressService` custom resource (CR) to manage egress traffic for pods behind a load balancer service. This is available as a Technology Preview feature.

You can use the `EgressService` CR to manage egress traffic in the following ways:

* Assign the load balancer service's IP address as the source IP address of egress traffic for pods behind the load balancer service.

* Assign the egress traffic for pods behind a load balancer to a different network than the default node network.

For more information, see xref:../networking/ovn_kubernetes_network_provider/configuring-egress-traffic-for-vrf-loadbalancer-services#configuring-egress-traffic-loadbalancer-services[Configuring an egress service].

[id="ocp-4-14-networking-metallb-vrf"]
==== Support for VRF specification in MetalLB's `BGPPeer` resource (Technology Preview)

> With this update, you can specify a Virtual Routing and Forwarding (VRF) instance in a `BGPPeer` custom resource. MetalLB can advertise services and make routing decisions based on the routing information in the VRF. This is available as a Technology Preview feature.

Reviewer 2: ```suggestion
With this update, you can specify a Virtual Routing and Forwarding (VRF) instance in a `BGPPeer` custom resource. MetalLB can advertise services via the interfaces belonging to the VRF. This is available as a Technology Preview feature.
```
